mesheryctl/internal/cli/root/system/start.go

```diff
@@ -42,10 +45,10 @@ var startCmd = &cobra.Command{
 	Short: "Start Meshery",
 	Long:  `Run 'docker-compose' to start Meshery and each of its service mesh adapters.`,
 	Args:  cobra.NoArgs,
-	PreRunE: func(cmd *cobra.Command, args []string) error {
-		//Check prerequisite
-		return utils.PreReqCheck(cmd.Use)
-	},
+	//PreRunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Why is Pre-check removed? @anirudhjain75

Reply:
Because moving forward we are looking for multiple platform support @leecalcote can brief you more on why I removed it

Reply:
okay, @leecalcote please intervene. I think the PreCheck is necessary to check for docker-compose availability.

Reply:
As we build in support for `platform`, we'll want to ensure that a prerequisite check is being run for docker-compose/docker anytime that `mesheryctl` is invoking docker-compose/docker. 


We'll need a switch/case statement to ensure that "docker" is being handled for contexts of those platform types.

Reply:
Sure. I'll make the change and push it in about 10 hours from now

Reply:
Made the changes